sql/src/main/java/io/crate/cluster/gracefulstop/DecommissioningService.java

```diff
@@ -324,4 +336,8 @@ private void setGracefulStopForce(boolean forceStop) {
     private void setDataAvailability(DataAvailability dataAvailability) {
         this.dataAvailability = dataAvailability;
     }
+
+    private boolean hasNoneDataAvailability() {
```

Comment:
nitpicking: I think having a method for this is a bit too zealous :)

Reply:
well the reason for having it here is readability I assume - to add a bit of semantic to the condition inside - but I think the condition itself is readable enough (reusability must've been low on the reasons list as it's just evaluating a condition, or was it reusability?)

Reply:
Fully agree with the reasoning of andrei. I'd even argue that the indirection caused by the method decreases readability in this case. But that's personal opinion and nitpicking. So I also agree on keeping it if you feel like it.

> it'll be inlined at runtime anyway

Not necessarily, I think that's only done if it is detected as a hotspot. Either way. Hardly matters for this functionality.

Reply:
k, I'm convinced and I'll change it